### ISSUE TITLE
`os_name()` only works once on Windows

### ISSUE THREAD

**Issue by Gobidev** [Macchina-CLI/libmacchina#139]
Calling `os_name()` on Windows only returns the correct value the first time it is called in a program. Using a new readout does not help.

The follwing program
```rust
use libmacchina::*;

fn main() {
    use libmacchina::traits::{GeneralReadout as _};

    let general_readout = GeneralReadout::new();
    let _ = dbg!(general_readout.os_name());
    let _ = dbg!(general_readout.os_name());

    let general_readout = GeneralReadout::new();
    let _ = dbg!(general_readout.os_name());
}
```

produces this output:
```
[src\main.rs:7] general_readout.os_name() = Ok(
    "Microsoft Windows 11 Pro",
)
[src\main.rs:8] general_readout.os_name() = Err(
    Other(
        "HRESULT Call failed with: 0x80010119",
    ),
)
[src\main.rs:11] general_readout.os_name() = Err(
    Other(
        "HRESULT Call failed with: 0x80010119",
    ),
)
```
The expected output would be:
```
[src\main.rs:7] general_readout.os_name() = Ok(
    "Microsoft Windows 11 Pro",
)
[src\main.rs:8] general_readout.os_name() = Ok(
    "Microsoft Windows 11 Pro",
)
[src\main.rs:11] general_readout.os_name() = Ok(
    "Microsoft Windows 11 Pro",
)
```

**Comment by FantasyTeddy** [comment]
Good catch! I can reproduce this error and it seems to be related to this issue: ohadravid/wmi-rs#37
Let me see if I can fix this.

**Comment by Gobidev** [comment]
Fixed in #143
